src/util/cmdline.h

```diff
@@ -20,6 +20,58 @@ Author: Daniel Kroening, kroening@kroening.com
 class cmdlinet
 {
 public:
+  /// Parses a commandline according to a specification given in optstring.
+  /// \param argc How many arguments there are.
+  /// \param argv An array of C strings.
+  ///             The 0th element is assumed to be the name of the command as
+  ///             it was invoked (e.g. /usr/bin/cmake) and is ignored. It is
+  ///             further assumed the array holds argc+1 elements with the C
+  ///             string at index argc being a terminating null pointer.
+  ///             This parameter is parsed based on optstring.
```

Comment:
Again, `\p optstring`, and s/parameter/argument/ ("argv" is a parameter, but what gets parsed is the actual argument).